Java/Assignment_3.java

```diff
@@ -0,0 +1,26 @@
+package com.company;
+
+import com.company.code.TheCode;
+
+import java.io.IOException;
+import java.net.InetAddress;
+
+public class Main {
+    public static void main(String[] args) throws IOException {
+        long start=System.currentTimeMillis();
```

Comment:
Give proper variable names, like in case of start, end, time

Reply:
Done !